app/controllers/Application.scala

```diff
@@ -61,9 +61,11 @@ class Application @Inject()(data: DataHelper,
       val future = actions.collect(filter, categoryArray, projectsPerPage, offset, ordering)
       val projects = this.service.await(future).get
 
+      val total = this.service.await(this.service.getActions(classOf[ProjectActions]).count()).get
```

Comment:
Perhaps name this something other than just `total`? It's unclear what it is without seeing where it's used.